docs/upgrade.md

```diff
@@ -116,6 +116,31 @@ metadata:
   name: clusterID-observability-bundle-user-values
   namespace: clusterID
 ```
+#### VPA users
+
+If you are using `VerticalPodAutoscaler` CRs to scale your Prometheus resources vertically, you will have to adjust your VerticalPodAutoscaler resources.
+
+This is needed because the scale subresouce was added to the Prometheus CRs in latest Prometheus Operator release.
```

Comment:
I would rather have something like this

```suggestion
From version vX.Y.Z field of `VerticalPodAutoscaler` CR's `targetRef` field needs to be updated with

```diff
    targetRef:
-    apiVersion: apps/v1
-    kind: StatefulSet
+    apiVersion: monitoring.coreos.com/v1
+    kind: Prometheus
     name: prometheus-whatever-name
```

This is needed because the scale subresouce was added to the Prometheus CRs in latest Prometheus Operator release.
```

Reply:
I applied your suggestion, can you take another look?

Reply:
Looks good